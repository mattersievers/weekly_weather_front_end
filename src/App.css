.App {
  text-align: center;
  display: flex;
  justify-content: center;
  align-items: center;
  min-height: 100vh;
}

.weatherContainer{
  display: flex;
  align-items: center;
  justify-content: center;
  border: 1px solid black;
}

.weatherDayContainer0{
  background-color: lightgray;
}
.dayContainer{
  margin: 1vw;
  padding: 1vw;
}

.minTemp{
  color: rgb(100, 100, 100);
}