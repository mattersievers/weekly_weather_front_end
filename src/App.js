import './App.css';
import DayWeather from "./components/DayWeather";

function App() {
  return (
    <div className="App">
     <DayWeather/>
    </div>
  );
}

export default App;
