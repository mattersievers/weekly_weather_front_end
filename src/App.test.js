import React from "react";
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import App from './App';

afterEach(cleanup);

describe('App component', () => {
  it('App renders', () => {
      render(<App />);
  });
  it('matches snapshot DOM node structure', () => {
    const {asFragment} = render(<App />);
    expect (asFragment()).toMatchSnapshot();
  })
  
})