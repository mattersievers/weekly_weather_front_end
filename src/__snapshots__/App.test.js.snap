// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div>
    Loading...
  </div>
</DocumentFragment>
`;
